.npmignore

```diff
@@ -1,45 +1,17 @@
-ccxt.egg-info/
-build/
-examples/
-node_modules/
-python/
-php/
-cs/
-c#/
-tmp/
-vendor/
-wiki/
-.tox/
-.github/
-.DS_Store
-.nyc_output
-.travis.yml
-appveyor.yml
-*.php
-*.cs
-*.py
-*.pyc
-ccxt.sublime-workspace
-composer.*
-config.js
-coverage
-exchanges.json
-ISSUE_TEMPLATE.md
-keys.local.json
-keys.json
-npm-debug.log
-travis-keys.sh
-run-tests.js
-Dockerfile
-docker-compose.yml
-phpunit.xml.dist
-setup.cfg
-exchanges.cfg
+# Initally ignore all files
+*
 
-# TS sources and JS tests
-/ts/
-/js/src/test/
-/js/src/pro/test/
+# Files to include
+!package.json
+!package-lock.json
+!postinstall.js
+!LICENSE.txt
+!README.md
 
-# Ignore cjs bundle
-dist/ccxt.bundle.cjs
\ No newline at end of file
+# Folders to include
+!js/**/*
+!dist/**/*
+
+# Ignore tests
+js/src/test/**/*
+js/src/pro/test/**/*
```

Comment:
I think we're missing ignoring the bundled version here
![image](https://github.com/ccxt/ccxt/assets/43336371/70f3f336-12ec-46ae-8a0b-74d100c68d7b)